snooty/util.py

```diff
@@ -50,7 +50,7 @@ def get_files(root: PurePath, extensions: Container[str]) -> Iterator[Path]:
     for base, dirs, files in os.walk(root):
         for name in files:
             ext = os.path.splitext(name)[1]
-
+            print(base, " ", ext)
```

Comment:
We definitely don't want to be printing in a utility function like this

Reply:
ack! yes